### DISCUSSION TITLE
Slower fan startup

### DISCUSSION THREAD

**Discussion by nathanielstenzel** [aegean-odyssey/mpmd_marlin_1.1.x#93] [Ideas]
My hotend fan is extremely noisy when it starts up. Would it be possible to start it at 50% power for a few seconds each time it starts up? I believe this is possible via a Configuration.h setting or the advanced configuration file. I forget which. I am not the only one with this issue, am I?

**Comment by PurpleHullPeas** [comment]
You can do it in your Start Gcode. See this issue: https://github.com/aegean-odyssey/mpmd_marlin_1.1.x/issues/41

**Comment by nathanielstenzel** [comment]
> ;Generated with Cura_SteamEngine 4.12.1
> M140 S85
> M105
> M190 S85
> M104 S226
> M105
> M109 S226
> M82 ;absolute extrusion mode
> ;MPMD V1 Basic Calibration Tutorial: 

The MPMD V1 line is the start of the gcode that I can set. Unless I can make that different, I would still be stuck with my fan trying to tear itself up.

**Comment by nathanielstenzel** [comment]
I think I figured it out for Cura but it is a pain in the butt and probably beyond what many could figure out. This needs to be before the non-comments in the start gcode setting for Cura.
```
M106 P0 S128
M104 {material_print_temperature}
M105
M109 {material_print_temperature}
M106 P0 S256

```

**Reply by PurpleHullPeas** [reply]
I linked you to an issue where I already figured it out for Cura and pasted example Start Gcode.
